css/properties/transition-behavior.json

```diff
@@ -34,6 +34,77 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "transitions_content-visibility": {
```

Comment:
Optional suggestion: what if we named these so that they were more explicitly reflecting the the thing it's duplicating? That is, these keys would be something like:

- `is_transitionable_content-visibility`
- `is_transitionable_display`

Or `transitionaable_`? Anything to make these show up together when searched would be nice.

Reply:
Thanks both, renamed in 8d786d60c.